apps/web/app/api/v1/js/lib/surveys.ts

```diff
@@ -21,7 +21,7 @@ export const getSyncSurveysCached = (environmentId: string, person: TPerson) =>
     async () => {
       return await getSyncSurveys(environmentId, person);
     },
-    [`getSyncSurveysCached-${environmentId}-${person.id}`],
+    [`getSyncSurveysCached-${environmentId}`],
```

Comment:
when the person is still passed to this function we also still need to personId in the cache key because if not the cache will return the same result for every person.